run.go

```diff
@@ -14,36 +15,59 @@ import (
 	"runtime"
 )
 
+const PLAYWRIGHT_CLI_VERSION = "0.170.0-next.1608058598043"
+
 type playwrightDriver struct {
-	driverName, driverFolder, driverPath, version string
+	driverDirectory, driverBinaryLocation, version string
 }
 
-func newDriver() (*playwrightDriver, error) {
-	cwd, err := os.Getwd()
-	if err != nil {
-		return nil, fmt.Errorf("could not get cwd: %w", err)
+func newDriver(options *RunOptions) (*playwrightDriver, error) {
+	baseDriverDirectory := ""
+	if options != nil {
+		baseDriverDirectory = options.DriverDirectory
+	}
+	if baseDriverDirectory == "" {
+		var err error
+		baseDriverDirectory, err = getDefaultCacheDirectory()
+		if err != nil {
+			return nil, fmt.Errorf("could not get default cache directory: %v", err)
+		}
 	}
-	driverFolder := filepath.Join(cwd, ".ms-playwright")
-	driverName := getDriverName()
-	driverPath := filepath.Join(driverFolder, driverName)
+	driverDirectory := filepath.Join(baseDriverDirectory, "playwright-go", PLAYWRIGHT_CLI_VERSION)
+	driverBinaryLocation := filepath.Join(driverDirectory, getDriverName())
 	return &playwrightDriver{
-		driverPath:   driverPath,
-		driverFolder: driverFolder,
-		driverName:   driverName,
-		version:      "0.170.0-next.1608058598043",
+		driverBinaryLocation: driverBinaryLocation,
+		driverDirectory:      driverDirectory,
+		version:              PLAYWRIGHT_CLI_VERSION,
 	}, nil
 }
 
+func getDefaultCacheDirectory() (string, error) {
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		return "", fmt.Errorf("could not get user home directory: %v", err)
+	}
+	switch runtime.GOOS {
+	case "windows":
+		return filepath.Join(userHomeDir, "AppData", "Local", "ms-playwright"), nil
```

Comment:
I'm not sure if we should store files in the same folder as playwright. It's cleaner to maintain a separate folder `playwright-go` on the root of the cache directory.

Reply:
I agree, lets maybe call it ms-playwright-go then so we don't "mess" up the original directory.

Reply:
I think `ms` stands for Microsoft :smile: I don't mind :+1:

Reply:
I think its fine because then the directories are directly next to each other.